OracleWebLogic/samples/12213-domain-wdt/README.md

```diff
@@ -0,0 +1,98 @@
+Example Image with a WLS Domain
+===============================
+This Dockerfile extends the Oracle WebLogic Server image by creating a sample WLS 12.2.1.3 domain and cluster. Utility scripts are copied into the image, enabling users to plug Node Manager automatically into the Administration Server running on another container.
+
+The Oracle WebLogic Deploy Tooling (WDT) automates domain creation and application deployment.  The tool creates the domain using a declarative, metadata model describing the domain and applications (with their dependent resources). 
+
+The Dockerfile uses the `createDomain` script from the WebLogic Deploy Tooling to create the domain from a text-based model file. More information about WDT is available in the README file for the WDT project in GitHub:
+
+`https://github.com/oracle/weblogic-deploy-tooling`
+
+### WDT Model File and Archive
+
+This sample includes a basic WDT model, `simple-topology.yaml`, that describes the intended configuration of the domain within the Docker image. WDT models can be created and modified using a text editor, following the format and rule described in the README file for the WDT project in GitHub.
+
+Another option is to use the WDT `discoverDomain` tool to create a model. This process is also described in the WDT project's README file. A user can use the tool to analyze an existing domain, and create a model based on its configuration. The user may choose to customize the model before using it to create a new Docker image.
+
+Domain creation may require the deployment of applications and libraries. This is accomplished by creating a ZIP archive with a specific structure, then referencing those items in the model. This sample creates and deploys a simple ZIP archive containing a small application WAR. That archive is built in the sample directory prior to creating the Docker image.
+
+When the WDT `discoverDomain` tool is used on an existing domain, a ZIP archive is created containing any necessary applications and libraries. The corresponding configuration for those applications and libraries is added to the model.
+
+### How to Build and Run
+
+**NOTE:** The image is based on a WebLogic Server image in the docker-images project: `oracle/weblogic:12.2.1.3-developer`. Build that image to your local repository before building this sample.
+
+The WebLogic Deploy Tool installer is required to build this image. Add `weblogic-deploy.zip` to the sample directory.
```

Comment:
This is open source and publicly accessible, so why don't you just pull it automatically using `curl` inside the `Dockerfile`?